db/schema.rb

```diff
@@ -22,4 +22,30 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "prizes", force: :cascade do |t|
+    t.bigint "wheel_id"
+    t.string "name"
+    t.integer "percentage"
+    t.string "image"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["wheel_id"], name: "index_prizes_on_wheel_id"
+  end
+
+  create_table "wheel_configurations", force: :cascade do |t|
+    t.bigint "wheel_id"
+    t.string "color_1"
+    t.string "color_2"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["wheel_id"], name: "index_wheel_configurations_on_wheel_id"
+  end
```

Comment:
You need to run the migrations to have the schema updated. It's still showing here with `color_1` instead of the new naming